src/components/layout-editor/layout-editor-control-resolver.tsx

```diff
@@ -49,22 +49,28 @@ export interface IResolverContext {
   selectedControls: string[];
 }
 
-export function getControlTypeName(cell): string {
-  const controlDefinition =
-    cell.controlType && findResolverByType(cell.controlType);
-
-  if (controlDefinition) {
-    return controlDefinition.typeName;
-  } else {
-    return "Unknown control";
-  }
+export function getCellCommonsAttrs(
```

Comment:
```suggestion
export function getCellCommonAttrs(
```

For updating the name, try using VS.Code's renaming feature, it will be a breeze.